docs/src/components/landing-page/TwitterCard.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div v-html="cardContent">
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { fabTwitter } from '@quasar/extras/fontawesome-v5'
+
+export default defineComponent({
+  name: 'ShowCaseCard',
+  props: {
+    cardContent: {
+      required: true,
+      type: String
+    }
+  },
+  setup () {
+    const scriptElement = document.createElement('script')
```

Comment:
I'm pretty sure you only need one of these, not one per twitter card
Move this code into landing index component setup, or even better into the index html template, as an async/defer script

Reply:
Strangely, when I move it to index template (as async or defered) or to the Index.vue or to the new component (TwitterShowcaseCards) which loads the TwitterCard compnent, only the first set of tweets load, the rest of the tweets in subsequent slides do not load. I'm yet to figure out what the issue is, I have however updated this PR (with the fixes), and with this issue commented, to give you a view.

I was also looking at  displaying a loading state when a tweet is being loaded. Coupled with the first issue, I may have to go with rendering with Js, which renders the tweets as a promise.

Reply:
Ok, let me know if you need help to understand what's going on there. Copy here the relevant resources you're using to add this feature, in you need help

Reply:
In order to embed the cards, one needs the script at `https://platform.twitter.com/widgets.js ` to be loaded in the head section of the page. Then add the code below to embed a tweet with id tweetId somewhere in the page. 
```
<blockquote class="twitter-tweet">
  <a :href="`https://twitter.com/marsdevs/status/${tweetId}`"></a>
</blockquote>
```
In order to inject the script at `https://platform.twitter.com/widgets.js` into the page, I have used the script below to do so 
```
scriptElement.setAttribute('src', 'https://platform.twitter.com/widgets.js')
scriptElement.setAttribute('charset', 'utf-8')
document.head.appendChild(scriptElement)
```
I added two extra components, one which displays the carousel (TwitterShowcaseCards.vue) and another which displays a twitter card (TwitterCard.vue).

I inject the head script from within  TwitterShowcaseCards.vue (this will now happen for every card again). But unfortunately only the first slide renders. If I inject the script from within TwitterCard.vue (where the script is called for every card), then the rest of the slides render again.

So at the moment, I have also tried injecting directly in the head tag of index.template.html, and the behaviour is exactly the same as injecting in TwitterShowcaseCards.vue (where only first slide renders).

Reply:
This is due to the twitter widget searching the tweets to embed only the first time you include it
You should use dynamic initialization upon slide change

![image](https://user-images.githubusercontent.com/10036108/140326761-05c4eefe-f039-4a33-9ef4-8bfe75c36ef4.png)